eCourse_backend/eCourse_backend/static/JS/user_data_print.js

```diff
@@ -0,0 +1,20 @@
+// print the content of the user information table
+function userInfoPrint()
+{
+    let divToPrint = document.getElementById('user_information');
+    let newWin = window.top.open();
+    newWin.document.open();
+    newWin.document.write('\
+        <html>\
+        <head>\
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">\
+        </head>\
+        <body onload="window.print()">' 
```

Comment:
Why do we need bootstrap here?

Reply:
Wanted to use it for the table, but didn't work -> removed